unit-tests/src/test/java/org/eclipse/collections/impl/list/mutable/FastListTest.java

```diff
@@ -803,7 +803,7 @@ public void addAtIndex()
     @Test
     public void testSubListSort()
     {
-        FastList<Integer> list = (FastList<Integer>) Interval.from(0).to(20).toList();
+        MutableList<Integer> list = Interval.from(0).to(20).toList();
```

Comment:
Same comment on explicit construction of `FastList`, since this is in `FastListTest`.